content/en/docs/chart_best_practices/templates.md

```diff
@@ -147,17 +158,23 @@ metadata:
 Both YAML and Helm Templates have comment markers.
 
 YAML comments:
+
 ```yaml
 # This is a comment
 type: sprocket
+foo: bar # This is a comment, bar isn't
 ```
 
-Template Comments:
+Helm template comments:
+
 ```yaml
+{{- /* This is a comment (with chomping to the right). */ -}}
+type: frobnitz
+
 {{- /*
-This is a comment.
+This is a multiline comment (with no chomping to the right).
 */}}
```

Comment:
why is there space between the comment syntax on line 174 and not here?

Reply:
It is required syntax, but not at all obvious and I bet more than I have run into this kind of syntax error. I added a commit warning about it.

>When using template comments, watch out for a common syntax error.
>
>Correct:
>
>```
>{{- /* Correct spacing with whitespace chomping*/ -}}
>{{/* Correct spacing without whitespace chomping */}}
>```
>
>Incorrect:
>
>```
>{{-/* Incorrect spacing with whitespace chomping */-}}
>{{ /* Incorrect spacing without whitespace chomping */ }}
>```